x86_energy_plugin.cpp

```diff
@@ -140,8 +140,8 @@ class x86_energy_measurement
 {
 public:
     x86_energy_measurement()
-    /* setting default value for reading_time */
-    : mreading_time(std::chrono::milliseconds(5)), is_thread(0)
+    /* setting default value for intervall_us */
+    : mintervall_us(std::chrono::microseconds(50000)), is_thread(0)
```

Comment:
Because the original name, that's used again, is interval_us and not intervall_ms.